src/pd/pd.rs

```diff
@@ -54,6 +54,7 @@ pub enum Task {
         peer: metapb::Peer,
         down_peers: Vec<pdpb::PeerStats>,
```

Comment:
Does `down_learners` need to be considered?

Reply:
fixed.